doc/misc/npm-faq.md

```diff
@@ -371,16 +355,12 @@ The npm open source project, The npm Registry, and [the community
 website](https://www.npmjs.com) are maintained and operated by the
 good folks at [npm, Inc.](http://www.npmjs.com)
 
-## I have a question or request not addressed here. Where should I put it?
+## I have a question or request not addressed here!
 
-Post an issue on the github project:
+Please post an issue on GitHub:
 
 * <https://github.com/npm/npm/issues>
 
-## Why does npm hate me?
```

Comment:
Sorry, can see why that was confusing. Meant to leave a note here and forgot. while it is light-hearted, it felt out of place when I read through it. And though the answer is positive, I don't really understand the point of the question. It seems to me it would imply that npm is broken and not working or in some other way "mistreating" the user in some way. Obviously a personal preference, and actually meant to leave it in and leave a note...Sorry for the confusion!